boards/samr34-xpro/Makefile.dep

```diff
@@ -0,0 +1,7 @@
+ifneq (,$(filter semtech-loramac,$(USEMODULE)))
```

Comment:
Like for the b-l072z-lrwan1, I would replace this block with the following:
```suggestion
ifneq (,$(filter netdev_default,$(USEMODULE)))
  USEMODULE += sx1276
endif
```

You may also want to add the board to the variable `BOARD_PROVIDES_NETIF` in `examples/default`

Reply:
@aabadie I'm confused. Should we use 
` USEMODULE += sx1276`
or
` USEMODULE += sx127x`
here ?

Reply:
nvm `grep` answers me.